### DISCUSSION TITLE
Supermercado

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#334] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/supermercado/

**Comment by brunoaugusto** [comment]
Filho de um parquímetro :joy:

**Comment by maryon-telvanni** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/tirinhas/supermercado/#comment_1160580110>):

> Filho de um parquímetro 😂

Às vezes é bem difícil de traduzir essas tirinhas norueguesas, mas essa tava bem na cara do que o cara tava chamando a máquina
